python/lsst/ctrl/mpexec/dotTools.py

```diff
@@ -49,32 +49,49 @@
 #  Local non-exported definitions --
 # ----------------------------------
 
-# Node styles indexed by node type.
-_STYLES = dict(
-    task=dict(shape="box", style="filled,bold", fillcolor="gray70"),
-    quantum=dict(shape="box", style="filled,bold", fillcolor="gray70"),
-    dsType=dict(shape="box", style="rounded,filled", fillcolor="gray90"),
-    dataset=dict(shape="box", style="rounded,filled", fillcolor="gray90"),
+# Attributes applied to directed graph objects.
+_ATTRIBS = dict(
+    defaultGraph=dict(splines="ortho", nodesep="0.5", ranksep="0.75", pad="0.5"),
+    defaultNode=dict(shape="box", fontname="Monospace", fontsize="14", margin="0.2,0.1", penwidth="3"),
+    defaultEdge=dict(color="black", arrowsize="1.5", penwidth="1.5"),
+    task=dict(style="filled", color="black", fillcolor="#B1F2EF"),
+    quantum=dict(style="filled", color="black", fillcolor="#B1F2EF"),
+    dsType=dict(style="rounded,filled,bold", color="#00BABC", fillcolor="#F5F5F5"),
+    dataset=dict(style="rounded,filled,bold", color="#00BABC", fillcolor="#F5F5F5"),
 )
 
 
+def _renderDefault(type: str, attribs: dict[str, str], file: io.TextIOBase) -> None:
+    """Set default attributes for a given type."""
+    default_attribs = ", ".join([f'{key}="{val}"' for key, val in attribs.items()])
+    print(f"{type} [{default_attribs}];", file=file)
+
+
 def _renderNode(file: io.TextIOBase, nodeName: str, style: str, labels: list[str]) -> None:
     """Render GV node"""
-    label = r"\n".join(labels)
-    attrib_dict = dict(_STYLES[style], label=label)
-    attrib = ", ".join([f'{key}="{val}"' for key, val in attrib_dict.items()])
+    label = r"</TD></TR><TR><TD>".join(labels)
```

Comment:
I'm not very familiar with how graphviz handles HTML, do you need to escape special HTML characters here (imagine someone adds an ampersand to task name)?

Reply:
According to the GraphViz docs (https://graphviz.org/doc/info/shapes.html#html), yes, HTML strings *will* be processed like HTML input. Do we think this will be a problem for us?

Reply:
I'm OK with HTML, but I think we need to do proper escaping of strings, just in case people start using fancy names. There is a `html.escape()` escape method that should take care of that.